src/utils/viewUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { addMinutes, format, formatDistanceToNow, parseISO } from 'date-fns';
+
+const getIsoToLocalDate = (iso: string): Date => {
+  const date = parseISO(iso);
+  console.log(date);
+  return addMinutes(date, -date.getTimezoneOffset());
+};
```

Comment:
I have noticed that all the dates that I am receiving from the backend are UTC, but because the ISO I am receiving does not contain a `Z` at the end, when parsing those ISOs with `Date()` or any date library, those UTC dates are automatically interpreted as local time. 

Kind of hacky, but the only way I found so far so solve this issue was to rest the timezone offtset and in that way it works. If you know a better solution, please share. If it was possible to send the ISO with the Z included at the end, that would be good too.

Reply:
I can implement it on the backend if you prefer. It would basically just consist of subclassing `datetime.datetime` and `datetime.date` and using those subclasses to parse the responses from the monitoring servers. Please confirm if you'd like this done.

Reply:
Leaving this one up to you. It is not the most ideal solution, but so far I was able to handle the problem on the front-end. If you decided to change it on the backend too, please let me know so I can update the front-end as well.

Reply:
Let's leave it on the front-end then.